contrib/copy-tex/README.md

```diff
@@ -0,0 +1,26 @@
+# Copy-tex extension
+
+This extension modifes the copy/paste behavior in any browser supporting the
+[Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardEvent)
+so that, when selecting and copying whole KaTeX-rendered elements, the text
+content of the resulting clipboard renders KaTeX elements as their LaTeX source
+surrounded by specified delimiters.  (The HTML content remains unchanged.)
```

Comment:
Maybe: "The rendered HTML content on the page remains unchanged."

Reply:
Actually, I meant that the HTML content of the copy/paste buffer remains unchanged.  Will revise.